nerdlandbot/translations/Translations.csv

```diff
@@ -156,4 +151,18 @@
 "empty_poncho_list","No Poncho pictures were found...","Er werden geen foto's van Poncho gevonden..."
 "poncho_pic_help","Post a random Poncho picture.","Post een willekeurige foto van Poncho."
 "poncho_hond","DOG DOG DOG","HOND HOND HOND"
-
+"recipe_1_question","the name of the dish","de naam van het gerecht in"
+"recipe_2_question","a link to the recipe (discord or external)","een link naar het recept (discord of extern) in"
+"recipe_3_question","how much croquettes you give the recipe","in hoeveel kroketten je het recept geeft"
+"recipe_4_question","any comments you might have on the recipe","alle opmerkingen in die je hebt over het recept"
+"recipe_5_question","your name","je naam in"
+"recipe_template","Please enter {0}.","Voer alsjeblieft {0}."
+"recipe_timeout","You ran out of time, please try again!","Je bent de tijd uit het oog verloren, probeer het opnieuw!"
+"recipe_int_error","Enter minutes, not words!","Voer minuten in, geen woorden!"
```

Comment:
I think "minutes" is copied from somewhere else, since you don't need minutes in the recipes?